velox/exec/tests/DriverTest.cpp

```diff
@@ -932,6 +932,18 @@ class ThrowOperator : public Operator {
   const ThrowNode::OperatorMethod throwingMethod_;
 };
 
+class ThrowIsBlockedOperator : public ThrowOperator {
```

Comment:
ThrowIsBlockedOperator name suggests that isBlocked method throws, but this is not the case.

Perhaps, rename to BlockedNoFutureOperator?